lib/services.template.ejs

```diff
@@ -180,6 +180,12 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' &&
     (action.sharedMethod.aliases || []).forEach(function(alias) {
       var aliasMethod = alias.split('.').join('$');
 
+      // this is to handle strong-remoting does not prepend prototype.
+      // back into prototype methods
+      if (action.sharedMethod.isStatic === false) {
+        aliasMethod = ['prototype', aliasMethod].join('$');
```

Comment:
I think aliases of prototype methods should be handled by strong-remoting. I thought we have already discussed this, was there a good reason why to keep the above workaround?